packages/contentstack/package.json

```diff
@@ -24,13 +24,13 @@
   "dependencies": {
     "@contentstack/cli-audit": "~1.6.3",
     "@contentstack/cli-auth": "~1.3.19",
-    "@contentstack/cli-cm-bootstrap": "~1.9.3",
+    "@contentstack/cli-cm-bootstrap": "~1.10.0",
     "@contentstack/cli-cm-branches": "~1.1.1",
     "@contentstack/cli-cm-bulk-publish": "~1.4.7",
     "@contentstack/cli-cm-export": "~1.11.5",
```

Comment:
Minor version bump required in seed command, also in documentation we need to add the --locale flag in seed command doc